src/lib/mina/actions/offchain-state.ts

```diff
@@ -33,7 +33,7 @@ import { ProvableType } from '../../provable/types/provable-intf.js';
 export { OffchainState, OffchainStateCommitments };
 
 // internal API
-export { OffchainField, OffchainMap };
+export { OffchainField, OffchainMap, OffchainStateInstance };
```

Comment:
This only exports the type internally, do you want to export it to users too?

Reply:
Nope, users don't need this, but exporting it internally lets TS imply the type when used in a smart contract.  Thus, we don't have to use `any`.